xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-security/xwiki-platform-extension-security-api/src/main/java/org/xwiki/extension/security/ExtensionSecurityScriptService.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.extension.security;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.apache.solr.client.solrj.SolrServerException;
+import org.slf4j.Logger;
+import org.xwiki.component.annotation.Component;
+import org.xwiki.extension.security.internal.ExtensionSecuritySolrClient;
+import org.xwiki.script.service.ScriptService;
+
+import static org.apache.commons.lang.exception.ExceptionUtils.getRootCauseMessage;
+
+/**
+ * Script service for the extension security extension. Gives access to the configuration.
+ *
+ * @version $Id$
+ * @since 15.5RC1
+ */
+@Component
+@Singleton
+@Named("extensionSecurity")
```

Comment:
Feels like it should be `extension.security` knowing that there's already various `extension.xx` script services.

Reply:
+1 for sub script service, but this security thing is supposed to be about indexed extensions, and we have a `extension.index` SS, so I would see the entry point of this new one more as `extension.index.security`